libtsuba/include/tsuba/GR.h

```diff
@@ -0,0 +1,38 @@
+#ifndef KATANA_LIBTSUBA_TSUBA_GR_H_
```

Comment:
I think it might be better to name this "topology" something rather than "gr" as the latter is a little cryptic.